charts/gardener/provider-local/extension/doc.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate ../../../../hack/generate-controller-registration.sh provider-local . ../../../../VERSION ../../../../example/provider-local/base/controller-registration.yaml DNSProvider:local DNSRecord:local ControlPlane:local Infrastructure:local Network:local OperatingSystemConfig:local Worker:local
```

Comment:
Is the pipeline configuration adapted to run generation on release and to commit also the generated files? (Similar to what we have for the extension repos)
Because on release the VERSION file will be changed and `make generate` will be required.

Reply:
Good point, not yet, we will take a look, thanks.